service/src/test/java/helpers/TestUtils.java

```diff
@@ -0,0 +1,168 @@
+package helpers;
+
+import com.myhome.domain.Amenity;
+import com.myhome.domain.Community;
+import com.myhome.domain.CommunityHouse;
+import com.myhome.domain.HouseMember;
+import com.myhome.domain.User;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static helpers.TestUtils.General.generateUniqueId;
+import static helpers.TestUtils.CommunityHouseHelpers.getTestHouses;
+import static helpers.TestUtils.UserHelpers.getTestUsers;
+
+public class TestUtils {
+
+  public static class General {
```

Comment:
@jmprathab 
Are you fine with that nested static classes?
This will grow as hell and we will have a big util class :(

Reply:
I can create separate classes for each nested class but this does not completely solve the problem. The best way is not using '*Util' classes but in this case i don't know how to get rid of repeating methods in test classes.

Reply:
Staying copy pasted code is not a good idea, just like using util classes. I can't find the third solution of this

Reply:
May bi in fact repeating code is not so terrible problem we can leave it is it is

Reply:
Personally, I am fine with this approach. I do not know of any other way to solve the problem of repeated code. My vote is to merge this PR in.